### ISSUE TITLE
Unable to install on Linux Mint 17.3

### ISSUE THREAD

**Issue by doghousedean** [tingbot/tide-electron#57]
Unable to install as a dependency is missing, I have tried to install the dep with pip but had no luck

`sudo pip install cryptography`

Can anyone help me?

```
sudo dpkg -i Tide-0.3.5.deb 
Selecting previously unselected package tide.
(Reading database ... 271388 files and directories currently installed.)
Preparing to unpack Tide-0.3.5.deb ...
Unpacking tide (0.3.5-99) ...
dpkg: dependency problems prevent configuration of tide:
 tide depends on python-cryptography; however:
  Package python-cryptography is not installed.

dpkg: error processing package tide (--install):
 dependency problems - leaving unconfigured
Processing triggers for desktop-file-utils (0.22-1ubuntu1) ...
Processing triggers for mime-support (3.54ubuntu1.1) ...
Processing triggers for hicolor-icon-theme (0.13-1) ...
Errors were encountered while processing:
 tide
```

**Comment by doghousedean** [comment]
I used the source code to compile it and was able to run it.

I am not familiar with node.js/npm to fix the problem directly

**Comment by Rob4001** [comment]
Hey there,

This seems to be a problem with the way we package up into a .deb file.
I'm not very knowledgeable with the Debian packaging system but from what I know the dpkg command does not install dependencies.

A quick Google says you can run "apt-get install -f" after the dpkg command to install the required dependencies

We should add this to the README or something to clear this up :)

**Comment by joerick** [comment]
Yeah, dpkg doesn't know how to find dependencies, just knows how to install, track, and complain about it! On Raspbian I've been using gdebi to install debs with dependencies, from the gdebi-core package. Maybe there's a standard tool for Debian/Ubuntu/Mint that does this in a nice way? If so, a readme update would be great. 

Joe

**Comment by doghousedean** [comment]
I find the dependencies are not being install and apt is removing the package.

```
[dean:~/Downloads] 7s 1 $ sudo apt-get install -f
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Correcting dependencies... Done
The following packages will be REMOVED
  tide
```

**Comment by davidudin** [comment]
The problem here seems to be that the Python sector of the Mint repositories does not include the python-cryptography package and depends on a package that python-cryptography is somehow in conflict with. I too tried to install the package with pip, but that doesn't communicate with debian dependency management world. When I attempted to install the package from the Ubuntu repository and install it with dpkg I got the conflict error.

I succesfully installed the python-cryptography on a raspbian system, but there apparently isn't a Tide .deb that is compatible with the particular arm architecture of the pi 3. Why does the package need to be architecture dependent?
